Core/GDCore/Project/Layer.cpp

```diff
@@ -13,7 +14,8 @@ namespace gd {
 Camera Layer::badCamera;
 Effect Layer::badEffect;

```

Comment:
You probably will move this to EffectsContainer, this static member will be useless in Layer.